app/models/template_input.rb

```diff
@@ -91,7 +99,7 @@ def resolved_value
 
   class UserInputResolver < InputResolver
     def ready?
-      !input_value.nil?
+      !input_value.value.nil?
```

Comment:
input_value can be also nil here which should result in not ready input, it would raise exception here